tests/regression.py

```diff
@@ -23,6 +23,9 @@
     # ],
     "/regression/e2e.test_keeper/test_zookeeper_operator_probes_workload*": [
         (Fail, "zookeeper liveness probe doesn't work, wait when https://github.com/pravega/zookeeper-operator/pull/476 will merge")
+        ],
+    "/regression/e2e.test_keeper/test_036. Check operator volume re-provisioning*": [
```

Comment:
It will pass test suite, even test failed. 
will need @sunsingerus to comment these lines for properly test running?


`"/regression/e2e.test_keeper/test_036*"`
should be enough